src/platform/platform_internal.h

```diff
@@ -662,10 +663,14 @@ CxPlatWorkerPoolGetEventQ(
     _In_ uint16_t Index // Into the config processor array
     );
 
+#ifdef _KERNEL_MODE // Not supported on kernel mode (yet)
+#define CxPlatDataPathProcessCqe(X) CXPLAT_FRE_ASSERT(FALSE)
```

Comment:
This means kernel mode doesn't process event (especially IO event)?
We will leave kernel mode datapath to be Wsk callback based?

Reply:
Correct. WSK stays as a callback.